nemo/collections/nlp/modules/common/token_classifier.py

```diff
@@ -0,0 +1,38 @@
+from torch import nn as nn
+
+from nemo.collections.common.parts import MultiLayerPerceptron, transformer_weights_init
+
+__all__ = ['TokenClassifier']
+
+ACT2FN = {"gelu": nn.functional.gelu, "relu": nn.functional.relu}
+
+class TokenClassifier(nn.Module):
```

Comment:
should this be a nemo.core.NeuralModule?

Reply:
done